test-cypress/cypress/commands/api/user.d.ts

```diff
@@ -35,5 +35,13 @@ declare namespace Cypress {
      * create user group and camera at the same time
      */
     apiCreateUserGroup(userName: string, group: string): Chainable<Element>;
+
```

Comment:
Was using this but no longer need too, should we keep it anyway for future use?

Reply:
If it works then we should.   As I've used it already in the other tests.